WhaleSpotting/react-app/src/styles/PageNav.scss

```diff
@@ -1,6 +1,33 @@
 @import "./Constants.scss";
 
 .page-nav {
-    display: flex;
+    display: inline-flex;
     justify-content: space-between;
+    width: 100%;
```

Comment:
not needed

Reply:
![image](https://user-images.githubusercontent.com/84719941/129167353-fa62c3c1-d2df-4f88-86ae-3ed42a723400.png)
It does affect when I change this

Reply:
Fair enough! I am wrong

Reply:
Fixed. It was coming with inline-flex